tests/auth.setup.ts

```diff
@@ -20,16 +20,21 @@ describe("Setup", async () => {
       process.env.USER1USERNAME,
       process.env.USER1PASSWORD
     );
-    // Example of how to add another user
-    // await logout(page)
-    // await logInWithUsernameAndPassword(
-    //     page,
-    //     process.env.USER2USERNAME,
-    //     process.env.USER2PASSWORD
-    // );
-    // Example of how to switch to said user
+  });
+  // Example of how to add another user
+  setup("Authenticate user 2", async ({ page }) => {
```

Comment:
@Andrewgdewar , do I need this second `setup` line?

When I try run a test I get:
```
Error: Project dependencies should have unique names, reading setup
```

Reply:
You likely could do this within the first setup, but it's okay if we separate it I suppose. 
We may also want to pass in the required ENV variables to make it obvious if we are missing something for each that way.